Taskfile.yml

```diff
@@ -60,16 +60,6 @@ tasks:
       # TODO : Check rule to be clarified with Prestashop, check commented in the meantime
       # - grep --color -r -P  '="{\$[^\s|]*}' alma/views/templates/* || exit 0 && >&2 echo "Variables in smarty templates should use |escape:'htmlall':'UTF-8'" && exit 1
 
-  lint:fix:
```

Comment:
I just noticed that we were pinged on this PR, as from your Slack message, I understand that this deletion (+ in the pre-commit file) is not expected ?

Reply:
I restore the lint:fix but commented the scripts remove-blank-line because it fails pre-commit and i can not push 
![image](https://github.com/alma/alma-installments-prestashop/assets/110386752/91b76d2b-ebfd-4e53-9b0f-b66e74acf434)

Reply:
And` lint:ci` failed in the ci but `lint` is ok on my pre commit

Reply:
OK, sorry for the trouble, I'll try to figure out why